cryptol-remote-api/python/tests/cryptol_eval/test-files/test-subdir/Bar.cry

```diff
@@ -0,0 +1,7 @@
+module Bar where
```

Comment:
It's a little unfortunate to need two copies of this file, but it's not really a big deal.

Reply:
Yah, the way we have testing set up currently means it's quite convenient to have distinct directories for the standard and eval server... perhaps theres a set of "common test cases" that should work the same for both and we could reduce some duplication that way?